scipy/signal/ltisys.py

```diff
@@ -2494,3 +2496,218 @@ def place_poles(A, B, poles, method="YT", rtol=1e-3, maxiter=30):
     full_state_feedback.nb_iter = nb_iter
 
     return full_state_feedback
+
+
+def controller_Hessenberg(A,B, C=None, tol=None):
+    r"""
+    Reduce the linear system ``AX + BU`` to the controller Hessenberg form
+    H, B_bar where P*A*P.T = H and P*B = Bbar with P being an orthogonnal
```

Comment:
Also please keep the format of: A single line being the short summary; an empty line; a longer summary. Rationale: the very first line ends up in the summaries, and if it's longer than one line, they get weird.